recipes/jsongrapher/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jsongrapher" %}
+{% set version = "2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/jsongrapher-{{ version }}.tar.gz
+  sha256: aaf0618ae8faf48456dda5b9425b93b31de91f270c81f0808e0397653c79a5ca
+
+build:
+  noarch: python
```

Comment:
See the documentation on `noarch: python` recipes and the use of the `python_min` jinja variable:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Reply:
Thank you.  I have updated like below and all tests have passed 
  host:
    - python {{ python_min }}

  run:
    - python >={{ python_min }}